.changeset/angry-trains-bake.md

```diff
@@ -0,0 +1,2 @@
+---
+---
```

Comment:
Fixes

```
🦋  error Some packages have been changed but no changesets were found. Run `changeset add` to resolve this error.
🦋  error If this change doesn't need a release, run `changeset add --empty`.
🦋  info NO packages to be bumped at patch
🦋  ---
🦋  info NO packages to be bumped at minor
🦋  ---
🦋  info NO packages to be bumped at major
```

Reply:
Do we need this error? They [don't recommend it](https://github.com/changesets/changesets/blob/main/docs/checking-for-changesets.md). Why do we have this log in a patch btw? Couldn't we have it in the `lint:changeset` job if it fails?

Reply:
Yeah I guess they want GitHub users to leverage the bot instead of a local CI script. We briefly tried it in BP repo but turned it off because of too much noise, if I remember correctly. The result is the same, the CI script is just a bit more tricky to configure (which we did anyway). Context:
- https://github.com/blockprotocol/blockprotocol/pull/579
- https://github.com/blockprotocol/blockprotocol/pull/586

I don't mind switching back to a bot (in bot repos) but not sure if it's is worth including into this PR.

Reply:
I mean – do we need to throw an error for empty changesets at all? Rather than the implementation.

And also – if we do want one, why do we need a custom patch rather than using the [status](https://github.com/changesets/changesets/blob/main/docs/command-line-options.md#status) command as a CI check?